code/modules/awaymissions/maploader/reader.dm

```diff
@@ -321,7 +325,7 @@ var/global/dmm_suite/preloader/_preloader = null
 /dmm_suite/preloader/New(list/the_attributes, path)
 	.=..()
 	if(!the_attributes.len)
-		Del()
+		qdel(src)

```

Comment:
The only Del() calls I wanna see in this codebase are in the garbage collection/qdel subsystem. Yes, this will add some extra overhead to accomplish the same function, but it means that every last thing will now be incorporated into qdel().
Unless a maintainer asks me to change this, I'm gonna leave it as is.